recipes/configs/eleuther_evaluation.yaml

```diff
@@ -31,6 +31,7 @@ seed: 217
 tasks: ["truthfulqa_mc2"]
 limit: null
 max_seq_length: 4096
+batch_size: 2
```

Comment:
?

Reply:
Yeah, sorry this should match the default